pom.xml

```diff
@@ -10,8 +10,8 @@
     <repositories>
         <repository>
             <id>github</id>
-            <name>GitHub Apache Maven Packages</name>
-            <url>https://maven.pkg.github.com/trjohnny/AerotrackModel</url>
+            <name>Aerotrack Model</name>
```

Comment:
si in realtà questa è per tutto non solo model ma vabbe non serve a niente il campo name